scripts/pwned-passwords.sh

```diff
@@ -6,7 +6,7 @@ submit_to_s3='false'
 pwned_directory="pwned_passwords"
 number_of_passwords=3000000
 pwned_url="https://downloads.pwnedpasswords.com/passwords/pwned-passwords-sha1-ordered-by-count-v8.7z"
```

Comment:
we can prob remove this old URL now

Reply:
Yeah I think I missed a few things here in my first pass, also need to fix the `cleanup` step to remove the new tmp directory instead of the 7z file.

Reply:
> we can prob remove this old URL now

Removed lingering references in 25d38da70.